src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -50,7 +52,9 @@ public AddCommand parse(String args) throws ParseException {
         Comment comment = ParserUtil.parseComment(argMultimap.getValue(PREFIX_COMMENT).orElse(EMPTY_COMMENT));
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
 
-        Person person = new Person(name, phone, email, address, comment, tagList);
+        // Parse VIP status; default to false if not provided
+        boolean isVip = Boolean.parseBoolean(argMultimap.getValue(PREFIX_VIP).orElse(DEFAULT_VIP));
```

Comment:
Minor nit: might've made more sense to put this like with the previous paragraph